docs/operations/security-overview.md

```diff
@@ -272,3 +272,19 @@ Cluster to deep storage:
 Cluster to client:
 1. Druid authenticates with the client based on the configured authenticator.
 2. Druid only performs actions when an authorizer grants permission. The default configuration is `allowAll authorizer`.
+
+## Reporting security issues
+
+The Apache Druid team takes security very seriously. If you find a potential security issue in Druid, such as a way to bypass the security mechanisms described earlier, please report this problem to [security@apache.org](mailto:security@apache.org). This is a private mailing list. Please send one plain text email for each vulnerability you are reporting.
```

Comment:
can you remove the link and just put email id as it is? docs checker seem to be failing here.